medusa/version_checker.py

```diff
@@ -89,6 +89,29 @@ def run(self, force=False):
 
         self.amActive = False
 
+    def run_in_docker(self):
+        """"
+        Check if medusa is run in a docker container.
+
+        If run in a container, we don't want to use the auto update feature, but just want to inform the user
+        there is an update available. The user can update through getting the latest docker tag.
+        """
+        path = '/proc/{pid}/cgroup'.format(pid=os.getpid())
+        try:
+            if not os.path.isfile(path):
+                return False
+
+            with open(path) as f:
+                for line in f:
+                    if re.match("\d+:[\w=]+:/docker(-[ce]e)?/\w+", line):
```

Comment:
Probably better to use `r` and and escape the slashes: `r'\d+:[\w=]+:\/docker(-[ce]e)?\/\w+'`